assets/js/app.js

```diff
@@ -0,0 +1,47 @@
+// Set Algolia options
+const options = {
+    appId: "UI9O9HIU4I",
+    apiKey: "6a380ea9bf30cec543bb36e2b3bdee01",
```

Comment:
should this be in git?

Reply:
@exdx this is the search API Key, algolia has a ADMIN API key that allows you to load indexes that they recommend keeping a secret. 

The cavate here is that anyone with the search API key can use it to hit up our API, and algolia has a free tier that allows for 1000 search for free. So we have to make sure that those 1000 searches are from visitors to our sites and not anyone playing around with the API, but I haven't found a elegant solution for this